dashboard/app/models/ability.rb

```diff
@@ -129,6 +130,7 @@ def initialize(user)
         can :manage, :pd_workshop_summary_report
         can :manage, :pd_teacher_attendance_report
         can :manage, Pd::TeacherApplication
+        can :manage, :pd_user_admin
```

Comment:
I don't understand.  My understanding is that this section of Ability.rb grants Workshop Admin permission to the pd_user_admin functionality and does NOT grant any permissions to Workshop Organizers:

`      if user.workshop_organizer?
        can :create, Pd::Workshop
        can [:read, :start, :end, :update, :destroy, :summary, :filter], Pd::Workshop, organizer_id: user.id
        can :manage_attendance, Pd::Workshop, organizer_id: user.id, ended_at: nil
        can :read, Pd::CourseFacilitator
        can :index, :workshop_organizer_survey_report
        can :read, :pd_workshop_summary_report
        can :read, :pd_teacher_attendance_report
      end

      if user.permission?(UserPermission::WORKSHOP_ADMIN)
        can :manage, Pd::Workshop
        can :manage, Pd::WorkshopMaterialOrder
        can :manage, Pd::CourseFacilitator
        can :manage, :workshop_organizer_survey_report
        can :manage, :pd_workshop_summary_report
        can :manage, :pd_teacher_attendance_report
        can :manage, Pd::TeacherApplication
        can :manage, :pd_user_admin
      end`